src/OrbitBase/File.cpp

```diff
@@ -221,6 +221,26 @@ ErrorMessageOr<void> MoveFile(const std::filesystem::path& from, const std::file
   return outcome::success();
 }
 
+ErrorMessageOr<void> RemoveFile(const std::filesystem::path& file_path) {
```

Comment:
Building on Henning's other comment, if I understand correctly `remove` also returns a `bool`. If I understand correctly, when the file didn't exists then it returns false without setting the error.

Reply:
right.

Reply:
done.